locales/de/zodiac.json

```diff
@@ -0,0 +1,62 @@
+{
+	"aries": {
+		"name": "Widder",
+		"element": 0,
+		"stone": "Blutstein"
```

Comment:
Is it the right translation?

- https://de.wikipedia.org/wiki/Heliotrop_(Mineral)
- https://en.wikipedia.org/wiki/Heliotrope_(mineral)

Reply:
"Heliotrop" is the correct word.  

I think the guy just translated from "bloodstone" but the stone in question is heliotrope which is called "heliotrop" in german

Reply:
> "Heliotrop" is the correct word.
> 
> I think the guy just translated from "bloodstone" but the stone in question is heliotrope which is called "heliotrop" in german

You're right, but now I know.